Doc 1872/changelog (#674)
swan-io/docs: changelog/2026/2026-07-30.mdx

---
title: 30 July 2026
type: release
---

## Improvements

> 🔄 [Incoming Verification of Payee (VoP) requests](/topics/payments/credit-transfers/sepa/vop/incoming-vop-requests/#verification-outcomes) for accounts in `Closing` status now go through standard matching, returning Match, Close match, or No match. Previously, these requests returned Impossible to match, blocking senders at the verification step. This aligns VoP with incoming credit transfers, which `Closing` accounts [can already accept to help settle outstanding debts](/changelog/18-june-2026). Requests for accounts in `Closed` or `Suspended` status still return Impossible to match.

Reviewer: ```suggestion
🔄 [Incoming Verification of Payee (VoP) requests](/topics/payments/credit-transfers/sepa/vop/incoming-vop-requests/#verification-outcomes) for accounts in `Closing` status now go through standard matching, returning Match, Close match, or No match. Previously, these requests returned Impossible to match, blocking senders at the verification step. This aligns VoP with incoming credit transfers, where `Closing` accounts[already accept transfers to settle outstanding debts](/changelog/18-june-2026). Requests for accounts in `Closed` or `Suspended` status still return Impossible to match.
```

Reviewer: Pronoun mismatch and simplified the second part for readability.

## API updates

### Upcoming breaking changes

🛡️ Starting **31 August 2026**, the [`addReceivedSepaDirectDebitB2bMandate`](/topics/payments/direct-debit/guide-add-mandate) mutation enforces SEPA format validation on two fields:

> - `creditorIdentifier`: must follow the SEPA Creditor Identifier structure, including a valid country code, check digits, creditor business code, and country-specific identifier.

Reviewer: Can link to this: https://docs.swan.io/topics/merchants/online/sdd/#sci

Also, align the language so it's consistent, i.e., ISO two-letter country code, two-digit checksum, etc., etc.

- `mandateReference`: must be a valid SEPA mandate reference of up to 35 characters.

> This prevents creating mandates with incorrect data, which later cause `MandateInvalid` rejections on outgoing SEPA Direct Debit B2B transactions.

Reviewer: Should this be: 

which later cause MandateInvalid rejections on received (incoming) SEPA Direct Debit B2B transactions?

Since we're talking about the received SDD mandate mutation?

Reviewer 2: Good question! The mandate is received, but the resulting transactions are outgoing in our model. The [direct debit page](https://docs.swan.io/topics/payments/direct-debit/) frames it as Swan receiving instructions for an outgoing direct debit, and a rejected one creates an SDD-Out transaction (with "invalid mandate" as the example). We should keep "outgoing" for consistency.
